client/state/sites/reducer.js

```diff
@@ -40,6 +41,14 @@ const VALID_SITE_KEYS = Object.keys( sitesSchema.patternProperties[ '^\\d+$' ].p
  */
 export function items( state = {}, action ) {
 	switch ( action.type ) {
+		case WORDADS_SITE_APPROVE_REQUEST_SUCCESS:
```

Comment:
It's just a little odd since the interesting state information is being set on sites, rather than in this tree. I think it's ok to leave this here if we think we'll be storing more information here in the future, otherwise we can have a follow up PR to move this to a more appropriate location.